server/src/main/java/io/kroki/server/Server.java

```diff
@@ -68,7 +69,14 @@ public void start(Promise<Void> startPromise) {
   }
 
   static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer>> listenHandler) {
-    HttpServer server = vertx.createHttpServer();
+    Integer maxInitialLineLength = config.getInteger("KROKI_MAX_LINE_LENGTH");
+    HttpServerOptions serverOptions = new HttpServerOptions();
+    if (maxInitialLineLength != null) {
+      logger.info("KROKI_MAX_LINE_LENGTH: {}", maxInitialLineLength);
```

Comment:
It could be useful to add a _debug_ log with the whole configuration but I don't think we should log individual option using info level.
Could you please remove this line?

```suggestion
```

Reply:
Sure. I think logging of config is good to be able to collect config from many containers and reported by the log indexer but as you mention that should in that case ber done for the whole configuration.

And maybe not needed for this app.